tools/docker/syzbot/Dockerfile

```diff
@@ -29,25 +29,20 @@ RUN test "$(uname -m)" != x86_64 && exit 0 || \
 RUN curl https://dl.google.com/go/go1.22.7.linux-$(uname -m | sed 's/aarch64/arm64/;s/x86_64/amd64/').tar.gz | tar -C /usr/local -xz
 ENV PATH /usr/local/go/bin:$PATH
 
-# The default clang-14 is too old, install the latest one.
 RUN apt-get install -y -q gnupg software-properties-common apt-transport-https
+# The default clang-14 is too old, install the latest one.
+# See https://kernel.org/pub/tools/llvm.
+ARG LLVM_VERSION=llvm-15.0.7-x86_64
```

Comment:
https://github.com/google/syzkaller/blob/master/tools/docker/README.md#syzbot-image

I don't think it'll work on arm.\
Current Dockerfile version is multi-arch.

Reply:
Thanks for the catch! There are aarch64, x86_64 arch available LLVM on https://mirrors.edge.kernel.org/pub/tools/llvm/. Are these 2 arch enough for syzbot? If yes, then I will update the PR with `uname -m` condition

Reply:
>Are these 2 arch enough for syzbot?

Yes, we currently use only aarch64 and x86_64.